pluralize.js

```diff
@@ -222,6 +222,27 @@
     pluralRules.push([sanitizeRule(rule), replacement]);
   };
 
+  /**
+   * Remove a pluralization rule from the collection.
+   *
+   * @param {(string|RegExp)} rule
+   */
+  pluralize.removePluralRule = function (rule) {
+    var check = sanitizeRule(rule).toString();
+    var index = null;
+
+    for (var i = 0; i < pluralRules.length; i = i + 1) {
```

Comment:
I'd refactor this to just splice and return when found instead of needing to store an index in a variable.

Reply:
done 😄